models/token.js

```diff
@@ -9,12 +9,9 @@ const MODEL = {
         .integer()
         .positive(),
 
-    uuid: Joi
+    value: Joi
         .string()
```

Comment:
Done! Went with 40 characters since that's what github uses, and 36^40 is a pretty large number.